explorer.js

```diff
@@ -98,6 +98,12 @@ function qs(key) {
 	return match && decodeURIComponent(match[1].replace(/\+/g, " "));
 }
 
+function sanitizeString(str) {
+	const tmpElement = document.createElement("div");
+	tmpElement.innerText = str;
+	return tmpElement.innerHTML;
+}
```

Comment:
In case you also wondered if you need to destroy the element: https://stackoverflow.com/questions/1847220/javascript-document-createelement-delete-domelement
Apparently you don't. I didn't found any function on MDN to do that anyway

Reply:
> In case you also wondered if you need to destroy the element: https://stackoverflow.com/questions/1847220/javascript-document-createelement-delete-domelement
> Apparently you don't. I didn't found any function on MDN to do that anyway

Nice! Thanks :)